nav2_costmap_2d/CMakeLists.txt

```diff
@@ -101,19 +95,17 @@ target_link_libraries(layers
   nav2_costmap_2d_core
 )
 
-# TODO(bpwilcox): port costmap_2d_markers
-#add_executable(costmap_2d_markers src/costmap_2d_markers.cpp)
-#ament_target_dependencies(costmap_2d_markers ${${PROJECT_NAME}_EXPORTED_TARGETS} ${ament_EXPORTED_TARGETS})
-#target_link_libraries(costmap_2d_markers
-  #costmap_2d
-#)
-
-# TODO(bpwilcox): port nav2_costmap_2d_cloud
-#add_executable(nav2_costmap_2d_cloud src/costmap_2d_cloud.cpp)
-#ament_target_dependencies(nav2_costmap_2d_cloud ${${PROJECT_NAME}_EXPORTED_TARGETS} ${ament_EXPORTED_TARGETS})
-#target_link_libraries(nav2_costmap_2d_cloud
-  #nav2_costmap_2d_core
-#)
+add_executable(nav2_costmap_2d_markers src/costmap_2d_markers.cpp)
+ament_target_dependencies(nav2_costmap_2d_markers ${${PROJECT_NAME}_EXPORTED_TARGETS} ${ament_EXPORTED_TARGETS})
```

Comment:
I don't think this line is necessary. I expect nav2_costmap2_2d_markers would pick up all the additional dependencies from the nav2_costmap_2d_core dependency.

If it is necessary, I'd prefer to add the `${dependencies}` variable or list them out rather than use the `${ament_*}` variables. Those don't seem to get set if you go into the build directory and run make directly, which I find to be a useful ability for debugging and more rapid iteration when doing TDD.

Reply:
Yup, works fine. Pushed.